{
  "Header": {
    "Original": "CVE-2022-26488 (In Python before 3.10.3 on Windows, local users can gain privileges be ...)",
    "Line": 94589,
    "ID": "CVE-2022-26488",
    "Description": "(In Python before 3.10.3 on Windows, local users can gain privileges be ...)"
  },
  "Annotations": [
    {
      "Original": "- python3.10 \u003cnot-affected\u003e (Windows-specific)",
      "Line": 94590,
      "Type": "package",
      "Package": "python3.10",
      "Kind": "not-affected",
      "Description": "Windows-specific"
    },
    {
      "Original": "- python3.9 \u003cnot-affected\u003e (Windows-specific)",
      "Line": 94591,
      "Type": "package",
      "Package": "python3.9",
      "Kind": "not-affected",
      "Description": "Windows-specific"
    },
    {
      "Original": "- python3.7 \u003cnot-affected\u003e (Windows-specific)",
      "Line": 94592,
      "Type": "package",
      "Package": "python3.7",
      "Kind": "not-affected",
      "Description": "Windows-specific"
    },
    {
      "Original": "- python3.5 \u003cnot-affected\u003e (Windows-specific)",
      "Line": 94593,
      "Type": "package",
      "Package": "python3.5",
      "Kind": "not-affected",
      "Description": "Windows-specific"
    },
    {
      "Original": "- python2.7 \u003cnot-affected\u003e (Windows-specific)",
      "Line": 94594,
      "Type": "package",
      "Package": "python2.7",
      "Kind": "not-affected",
      "Description": "Windows-specific"
    }
  ]
}